.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,23 @@
+name: Feature request / Suggestion
+description: Suggest an idea for this project
+title: "[FEAT]: "
```

To keep the bracket tagging consistent with Bug and Crash we should titlecase Feat instead of uppercasing it.

```suggestion
title: "[Feat]: "
```

---

It's totally up to you if you want the brackets in the title. But you are right that we have labels for that ¯\_(ツ)_/¯

---

I just had some thoughts about this and think that omitting them is the cleaner way to go, especially as everything else mostly didn't carry bracket tags either and it worked quite well